Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesTUVWXYZ.swift

```diff
@@ -2337,19 +2366,17 @@ public struct TypeEffectSpecifiersSyntax: SyntaxProtocol, SyntaxHashable, _LeafS
     }
   }
   
-  /// ### Tokens
-  /// 
-  /// For syntax trees generated by the parser, this is guaranteed to be `throws`.
-  public var throwsSpecifier: TokenSyntax? {
```

Comment:
This needs a compatibility layer.

Reply:
It's handled via the extension of the `EffectSpecifiersSyntax` protocol.